packages/react-components/src/components/ChatMessage/ChatMessageComponentAsMessageBubble.tsx

```diff
@@ -303,6 +308,15 @@ const MessageBubble = (props: ChatMessageComponentAsMessageBubbleProps): JSX.Ele
   const myMessageContainerClasses = isBlockedMessage ? chatBlockedMyMessageClass : chatMyMessageClass;
   const messageContainerClasses = isBlockedMessage ? chatBlockedMessageClass : chatMessageClass;
   const rootLayout = _useChatMyMessageLayout();
+
+  const chatItemMessageContainerClasses = defaultChatItemMessageContainerStyles();
+  const chatItemMessageContainerClassName = mergeClasses(
+    // Need to merge the styles injected with props - Future PR
```

Comment:
Is there a task for this change?

Reply:
I belive so. Just linked it to you in teams.